### PR TITLE
Balder/balder/group commits to tls 2 remaining

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR
This is the last PR which will actually bundle commits together and add value.
I think it requires some extra thought...

### CODE REVIEW

File: searchlib/src/vespa/searchlib/transactionlog/domain.cpp [hunk 1]
```diff
@@ -311,10 +334,79 @@ waitPendingSync(vespalib::Monitor &syncMonitor, bool &pendingSync)
...
+        return 0ms;
+    }
+    return (vespalib::steady_clock::now() - _firstArrivalTime);
+}
+
+void
+Domain::commit(const Packet & packet, Writer::DoneCallback onDone) {
+    vespalib::MonitorGuard guard(_currentChunkMonitor);
+    if (! (_lastSerial < packet.range().from())) {
+        throw runtime_error(make_string("Incomming serial number(%ld) must be bigger than the last one (%ld).",
```
Comment (toregge) [inline searchlib/src/vespa/searchlib/transactionlog/domain.cpp:366]: Format string is wrong.  User PRIu64 for serial numbers.

File: searchlib/src/vespa/searchlib/transactionlog/domain.cpp [hunk 2]
```diff
@@ -363,8 +364,8 @@ void
 Domain::commit(const Packet & packet, Writer::DoneCallback onDone) {
     vespalib::MonitorGuard guard(_currentChunkMonitor);
     if (! (_lastSerial < packet.range().from())) {
-        throw runtime_error(make_string("Incomming serial number(%ld) must be bigger than the last one (%ld).",
-                                        packet.range().from(), _lastSerial));
+        throw runtime_error(fmt("Incomming serial number(%" PRIu64 ") must be bigger than the last one (%" PRIu64 ").",
```
Comment (toregge) [inline searchlib/src/vespa/searchlib/transactionlog/domain.cpp:367]: s/Incomming/Incoming/

File: searchlib/src/vespa/searchlib/transactionlog/domain.cpp (hunk 1 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/transactionlog/domain.cpp]: Fixed

File: searchlib/src/vespa/searchlib/transactionlog/domain.cpp (hunk 2 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/transactionlog/domain.cpp]: Fixed

**Review by toregge — APPROVED** [review]

**Comment by baldersheim** [comment]
Superceeded by #14304

**Pull request closed without merging**